modules/star/1.0/star.smk

```diff
@@ -27,7 +27,7 @@ CFG = op.setup_module(
 )
 
```

Comment:
@whelena did this change get reverted?

Reply:
the commenting out of include utils-1.0?

Reply:
I despise this interface. Now I can't even tell what the context of this question was.

Reply:
I revert the 1.0 version so it includes utils 1.0 and removed it from the 1.3 version since utils-2.0 is included from the project Snakefile